### PR TITLE
TELCODOCS-1803 - Moving ZTP docs > Edge computing

### PR DESCRIPTION
**Pull request by aireilly**
Moving "Clusters at the network far edge" out of "Scalability and performance" to a top level "Edge computing" section.

Merge alongside https://github.com/openshift/openshift-docs/pull/73896

Version(s):
enterprise-4.14+

Issue:
https://issues.redhat.com/browse/TELCODOCS-1803

Link to docs preview:
https://73895--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/ztp-deploying-far-edge-clusters-at-scale

QE review:
- [x] QE review not required

### CODE REVIEW

**Comment by aireilly** [comment]
@ahardin-rh Can you PTAL at this when you have a moment.

**Comment by nalhadef** [comment]
Please don't forget to squash your commits. I see two currently.

**Comment by nalhadef** [comment]
/lgtm

Thanks for squashing the commit so quickly!!

File: edge_computing/ztp-image-based-upgrade.adoc [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+* xref:../backup_and_restore/application_backup_and_restore/installing/oadp-installing-operator.adoc[Installing the OADP Operator]
+
+include::modules/ztp-image-based-upgrade-prep.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* link:https://www.redhat.com/en/blog/a-guide-to-creating-a-separate-disk-partition-at-installation-time[A Guide for Creating a Separate-disk Partition at Installation Time]
+
+* xref:../scalability_and_performance/scalability_and_performance/ztp_far_edge/ztp-image-based-upgrade#ztp-image-based-upgrade-shared-container-directory_ztp-image-based-upgrade[Sharing the container directory between `ostree` stateroots]
```
Comment (aireilly) [inline edge_computing/ztp-image-based-upgrade.adoc:46]: typo

**Pull request merged**